Improved: Grid and List (OFBIZ-11345)

PierreSmits opened the pull request:
According to the definition in widget-form.xsd the use of a combination of a form with type="list" is deprecated in favour of a grid.
Refactor various list forms into grids.
Refactor various list form references in screens.

modified in product component:
FeatureScreens.xml: from form ref to grid ref , additional cleanup
FeatureForms.xml: from form definition with list ref to grid definition with list ref, additional clean-up
PriceScreens.xml: from form ref to grid ref , additional cleanup
PriceForms.xml: from form definition with list ref to grid definition with list ref, additional clean-up
CatalogScreens.xml: from form ref to grid ref , additional cleanup
ProdCatalogForms.xml: from form definition with list ref to grid definition with list ref, additional clean-up

PierreSmits committed:
Imrpoved: Grid and List (OFBIZ-11345)

According to the definition in widget-form.xsd the use of a combination of a form with type="list" is deprecated in favour of a grid.
Refactor various list forms into grids.
Refactor various list form references in screens.

modified in product component:
FeatureScreens.xml: from form ref to grid ref , additional cleanup
FeatureForms.xml: from form definition with list ref to grid definition with list ref, additional clean-up

mbrohl commented:
Hi Pierre, can you please correct / amend the title and commit message ("Improved"). Thanks!

PierreSmits commented:
Thanks for the feedback, Michael.

PierreSmits committed:
Improved: List and Grid (OFBIZ-11345)

According to the definition in widget-form.xsd the use of a combination of a form with type="list" is deprecated in favour of a grid.
Refactor various list forms into grids.
Refactor various list form references in screens.

modified in product component:
PriceScreens.xml: from form ref to grid ref , additional cleanup
PriceForms.xml: from form definition with list ref to grid definition with list ref, additional clean-up

PierreSmits committed:
Improved: List and Grid (OFBIZ-11345)

According to the definition in widget-form.xsd the use of a combination of a form with type="list" is deprecated in favour of a grid.
Refactor various list forms into grids.
Refactor various list form references in screens.

modified in product component:
CatalogScreens.xml: from form ref to grid ref , additional cleanup
ProdCatalogForms.xml: from form definition with list ref to grid definition with list ref, additional clean-up
